src/org/redalert1741/steamworks/Robot.java

```diff
@@ -197,21 +199,28 @@ public void teleopPeriodic()
     	drive.swerve(-x,-y,-twist,-navx.getAngle(),fieldOrient);
     	///////////////////////////////////////////////////////////////////////////
     	//Climber
-    	if(driver.getTriggerAxis(Hand.kRight) > 0.1)
+    	if(driver.getTriggerAxis(Hand.kRight) > 0.1 || op.getTriggerAxis(Hand.kRight) > 0.1)
```

Comment:
In this particular case this is introducing some technical debt (the constant 0.1 is used twice, and is not configurable). Can you rewrite this so that the maximum is taken, compared with 0.1, and then assigned to the climb function?